container/sagemaker/nginx.conf.template

```diff
@@ -32,7 +32,7 @@ http {
     subrequest_output_buffer_size 100m;
 
     set $tfs_version %TFS_VERSION%;
-    set $default_tfs_model %TFS_DEFAULT_MODEL_NAME%;
+    %SET_DEFAULT_TFS_MODEL%
```

Comment:
just wondering why this variable naming convention is different from the ones above.

Reply:
With dynamic endpoint, they don't need a default model name, so I replaced the live above with a variable that can be set in serve.py; so this `set ...` line will only be set if dynamic endpoint is enabled.

Reply:
That is correct, however isn't the TFS_VERSION defined above also an assignment?

It looks like you replace with a default regardless.